Mathlib/LinearAlgebra/Matrix/PosDef.lean

```diff
@@ -165,4 +166,25 @@ def InnerProductSpace.ofMatrix {M : Matrix n n 𝕜} (hM : M.PosDef) :
   InnerProductSpace.ofCore _
 #align matrix.inner_product_space.of_matrix Matrix.InnerProductSpace.ofMatrix
 
+section ConjTransposeMul
+
+variable {m : Type _} [Fintype m]
+open BigOperators
```

Comment:
```suggestion
open scoped BigOperators
```

Reply:
Though you don't even use the notation, so I think you can drop this.

Reply:
Sure it is just for the goal display in infoview